tests/test_cnn/test_wrappers.py

```diff
@@ -374,3 +374,26 @@ def test_nn_op_forward_called():
         wrapper = Linear(3, 3)
         wrapper(x_normal)
         nn_module_forward.assert_called_with(x_normal)
+
+
+@patch('mmcv.cnn.bricks.wrappers.TORCH_VERSION', (1, 10))
+def test_fx_compatibility():
+    try:
+        from torch import fx
+
+        # ensure the fx trace can pass the network
+        for Net in (MaxPool2d, MaxPool3d):
+            net = Net(1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
+        for Net in (Linear, ):
+            net = Net(1, 1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
+        for Net in (Conv2d, ConvTranspose2d, Conv3d, ConvTranspose3d):
+            net = Net(1, 1, 1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
```

Comment:
Is it necessary to print the code?

Reply:
oh, that's used to supress the `unused variable` error.

Reply:
Oh, you can replace `gm_module = fx.symbolic_trace(net)` with `fx.symbolic_trace(net)`.

Reply:
Fixed in [fc883ec](https://github.com/open-mmlab/mmcv/pull/2883/commits/fc883ec85f8bbd08fe934d1fa19a22b7051b3c04).